components/camel-aws/src/test/java/org/apache/camel/component/aws/sqs/SqsComponentConfigurationTest.java

```diff
@@ -73,9 +73,9 @@ public void createEndpointWithMinimalArnConfiguration() throws Exception {
         
         ((JndiRegistry) ((PropertyPlaceholderDelegateRegistry) context.getRegistry()).getRegistry()).bind("amazonSQSClient", mock);
         SqsComponent component = new SqsComponent(context);
-        SqsEndpoint endpoint = (SqsEndpoint) component.createEndpoint("aws-sqs://arn:aws:sqs:region:account:MyQueue?amazonSQSClient=#amazonSQSClient&accessKey=xxx&secretKey=yyy");
+        SqsEndpoint endpoint = (SqsEndpoint) component.createEndpoint("aws-sqs://arn:aws:sqs:region:account:MyQueue?amazonSQSClient=#amazonSQSClient&accessKey=xxx&secretKey=yyy&region=US_EAST_2");
 
-        assertEquals("region", endpoint.getConfiguration().getRegion());
```

Comment:
Should not this assert be null before this change?

Reply:
Region value was taken from ARN which was wrong. Take a look at the source file changes.

Reply:
Why not? sqs://arn:aws:sqs:US_EAST_2:account:MyQueue?amazonSQSClient=#amazonSQSClient&accessKey=xxx&secretKey=yyy

Reply:
ARN is actually created like “arn:aws:sqs:us-east-2:account:MyQueue”. You can try creating a SQS Queue in AWS and check the syntax of ARN. It’s not the right place to get Region. Later the value is directly used in com.amazonaws API and gives problem in enum. Check the error message I posted in CAMEL-12647. Also aws-sns does not have this problem.

Reply:
In this case, region in ARN on sqs component possibly becomes irrelevant. Doesn't it? Maybe number of parts in ARN would require extre length checks and conditions control by this [line](https://github.com/apache/camel/pull/2424/files#diff-b27317b097ef229e2092bebe046882f9R60). What do you think @oscerd ?

Reply:
Yes. camel aws-sql component uses com.amazonaws.AmazonWebServiceClient and it does not require ARN to get instance of it. The required parameters are parsed by camel from the given ARN.

Reply:
The example of arn for an sqs endpoint is arn:aws:sqs:us-east-1:123456789012:MyQueue. The problem is that we are using the enum value of the Regions class, so probably we may need to double check if the region is specified as arn, then we need to manipulate it to use it and make it compliant to be used for the enum.  In my opinion the PR must be reviewed in this way.